src/node/delay.rs

```diff
@@ -267,8 +279,8 @@ struct DelayReader {
     last_written_index: Rc<Cell<Option<usize>>>,
     // local copy of shared `last_written_index` so as to avoid render ordering issues
     last_written_index_checked: Option<usize>,
-    // internal buffer used to compute output per channel at each frame
-    internal_buffer: Vec<f32>,
+    // internal buffer used to compute output delay infos for each sample
+    playback_infos: [PlaybackInfo; RENDER_QUANTUM_SIZE],
```

Comment:
Because you are not reusing any of these values between render quanta, this does not need to be an attribute of the DelayReader, but can just be a local variable in de `process` call. Saves a bit of memory

Reply:
ok done! 

This is a bit counter intuitive for me, as I have the impression that we reallocate the buffer each time, but I guess that's a JS deformation as we work only with the heap here...

Reply:
(did it on the `AudioBufferSourceNode` as well)

Reply:
Yes, that is probably very different in JS.
In Rust, an array `[T]` is always on the stack so there is no 're-allocation'
The only types on the heap in Rust are:
- collections (Vec, String, HashMap, etc - https://doc.rust-lang.org/stable/std/collections/ )
- pointer types (Box, Arc, Rc)